modules/installation-creating-gcp-private-dns.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-private-dns_{context}"]
+= Creating a private DNS zone in GCP
+
+You must configure a private DNS zone in Google Cloud Platform (GCP) for your
+{product-title} cluster to use. One way to create this component is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your GCP
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and associated subnets in GCP.
+
+.Procedure
+
+. Copy the template from the *Deployment Manager template for the private DNS*
+section of this topic and save it as `02_infra.py` on your computer. This
+template describes the networking and load balancing objects that your cluster
+requires.
+
+. Export the following variable required by the resource definition:
++
+----
+$ export CLUSTER_NETWORK=`gcloud compute networks describe ${INFRA_ID}-network --format json | jq -r .selfLink`
+----
+
+. Create a `02_dns.yaml` resource definition file:
++
+----
+$ cat <<EOF >02_dns.yaml
+imports:
+- path: 02_dns.py
+
+resources:
+- name: cluster-dns
+  type: 02_dns.py
+  properties:
+    infra_id: '${INFRA_ID}' <1>
+    cluster_domain: '${CLUSTER_NAME}.${BASE_DOMAIN}' <2>
+    cluster_network: '${CLUSTER_NETWORK}' <3>
+EOF
+----
+<1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<2> `cluster_domain` is the domain for the cluster, for example `openshift.example.com`.
+<3> `cluster_network` is the `selfLink` URL to the cluster network.
+
+. Create the deployment by using the `gcloud` CLI:
++
+----
+gcloud deployment-manager deployments create ${INFRA_ID}-dns --config 02_dns.yaml --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT}
+----
+
+. The templates do not create DNS entries due to limitations of Deployment
+Manager, so you must create them manually:
+
+.. Export the following variable:
++
+----
+$ export CLUSTER_IP=`gcloud compute addresses describe ${INFRA_ID}-cluster-public-ip --region=${REGION} --format json | jq -r .address`
```

Comment:
It's already defined in Creating load balancers in GCP section. So please remove it.